server/src/core/server/services/comments/pipeline/phases/external.ts

```diff
@@ -102,9 +102,10 @@ export interface ExternalModerationRequest {
   tenantDomain: string;
 }
 
+// BOOKMARK: (marcushaddon) do we need a new type to maintain type safety with joi?
 export type ExternalModerationResponse = Partial<
-  Pick<PhaseResult, "actions" | "status" | "tags">
->;
+  Pick<PhaseResult, "status" | "tags">
+> & { actions: PhaseResult["commentActions"] };
```

Comment:
This is because our public API for external moderation phases still accepts the `actions` field, but their type is now internally called `commentActions`